confgenerator/testdata/valid/linux/metrics-custom_collection_interval/input.yaml

```diff
@@ -12,32 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-logging:
-  receivers:
-    syslog:
-      type: files
-      include_paths:
-      - /var/log/messages
-      - /var/log/syslog
-  exporters:
-    google:
-      type: google_cloud_logging
-  service:
-    pipelines:
-      default_pipeline:
-        receivers: [syslog]
-        exporters: [google]
-
 metrics:
   receivers:
     hostmetrics:
       type: hostmetrics
       collection_interval: 30s
```

Comment:
Do we absolutely need to support this use case for GA?

Reply:
Discussed offline. Yes we do need to support this